deepfloyd_if/modules/stage_III_sd_x4.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, *args, model_kwargs=None, pil_img_size=1024, **kwargs):
                 ' Please run `pip install diffusers --upgrade`'
             )
 
-        model_id = os.path.join('stabilityai', self.dir_or_name)
+        model_id = 'stabilityai/' + self.dir_or_name
```

Comment:
Not sure if it's necessary, but might be safer to only make the change if the os is windows, like:
```
if os.name == 'nt':
    model_id = 'stabilityai/' + self.dir_or_name
else:
    model_id = os.path.join('stabilityai', self.dir_or_name)
```

Reply:
I believe it was a mistake to use os.path.join in the first place, since it's not really a path and huggingface expects strictly a `/`

Reply:
> I believe it was a mistake to use os.path.join in the first place, since it's not really a path and huggingface expects strictly a `/`

Ah true